### ISSUE TITLE
Sound goes to rear speakers on boot

### ISSUE THREAD

**Issue by wreyniers** [cpebit/aawireless-issues#71]
**Describe the bug**
When turning on the car, about half of the time when I start music either from Spotify or the car's built in radio the sound is routed to just the rear speakers in the car.  I also lose control over the volume. I can bring the audio back to the front speakers by setting a destination and as soon as I get voice guidance and the audio switches back to the music source the sound comes out of all the speakers again. 

What's interesting is that when the car is set to radio on start I can briefly hear audio from all speakers, then to the rear speakers, and then about half of the time it returns to all speakers. The other times it stays on the rear speakers until the aforementioned nav guidance is triggered.

It almost seems that during the AA handshake, AAW is sending some kind of routing initialization that only works sporadically. It would seem that if that back and forth wasn't triggered the audio would always be on all speakers and the volume control would remain functional as it always starts in the correct mode. Perhaps the handshake goes back and forth to quickly and the head unit hits some kind of race condition.

I have tried rebooting my head unit on the car and sometimes it resolves the issue but the most reliable method is to trigger guidance.

I have tried audio passthrough, all the different USB modes, auto resume on and off in AA to no avail.

I'd be happy to provide logs and such but haven't been able to figure out how to get those.

**Car (please complete the following information):**
 - Subaru Ascent

**Headunit (please complete the following information):**
 - Harmon Kardon

**Smartphone (please complete the following information):**
 - Device: Pixel 5
 - Android version: 11

**Additional context**
This behavior has persisted through v1.3, 1.4 and 1.5.

**Comment by Snirpo** [comment]
Supremely odd issue. You could try if the VAG fix changes anything in this regard. I think the speech audio channel is staying active for some reason. It also happens in full passthrough mode?

**Comment by IgwanaRob** [comment]
I've been dealing with this issue long before AA came into existence - it's PITA, and has gotten worse since getting a new phone with Android 11. I have a 2018 Outback, and like you say triggering the Nav voice tends to fix the audio routing/focus.

In my setup I use an NFC tag to run Tasker/Trigger tasks that turn on BT to connect to the car, and the cable connection streams audio along with the Android Auto signal, so there's a couple place for the audio routing to conflict. Now with AAW introduce I've only run into this once so far (only got it yesterday :p ), but the Nav voice trick worked exactly the same. 

There is an audio focus setting in the Dev settings of the AA app, as well as the one in the AAW app. One (or both) might work - I have had the chance to try playing around with it too much today, but I had no problems on my one trip this morning

**Comment by wreyniers** [comment]
@IgwanaRob I looked everywhere I can think to look for audio focus settings both in the AA developer settings and in AAW settings. Is there something specific I should be looking for that isn't called audio focus?

**Comment by fa1c0n64** [comment]
I've actually been dealing with what appears to be the same issue for some time now. About half the time when I connect whatever audio source I have playing in AA seems to only play through a few of the speakers and volume control, etc.  do not work.

I found, similar to you, that activating voice nav would fix it.  Once some nav commands were spoken the rest of the audio would start working as it should, all speakers, and I could control volume again.

Recently I discovered simply activating the voice command feature using the steering wheel button, then hitting the button again to cancel, it seems to fix the issue as well.

I did some testing over the last couple of days and I'm starting to think this is not an AAW issue though.  I plugged my phone directly in via USB, a different cable than the AAW device used, and noted I would still periodically get the audio issue and the same fixes worked.

Car: 2018 Ram 
Headunit : UConnect 8.4
Smartphone : Pixel 6 Pro/Android 12
